lib/miq_automation_engine/service_models/miq_ae_service_physical_server.rb

```diff
@@ -0,0 +1,11 @@
+module MiqAeMethodService
+  class MiqAeServicePhysicalServer < MiqAeServiceModelBase
+    expose :ext_management_system, :association => true
+    expose :physical_servers,      :association => true
```

Comment:
@AndreyMenezes I am not seeing where this relationship is defined in the active_record models.  https://github.com/ManageIQ/manageiq/blob/master/app/models/physical_server.rb.

Does it make sense that a physical_server instance would `have_many` physical_servers as a relationship?